### ISSUE TITLE
Bluetooth Crash

### ISSUE THREAD

**Issue by SaintPatrck** [nasa-jpl/osr-android-app#8]
Application crashes if bluetooth is not available on the device.

Reproduction steps:
- Run application on emulator
- Attempt to connect via bluetooth

Expected:
- Application gracefully handles connection failure and displays message to user.

Actual:
- Application crashes if bluetooth is not available.

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.Set android.bluetooth.BluetoothAdapter.getBondedDevices()' on a null object reference
        at ejunkins.rovercontroller.MainActivity.onClick(MainActivity.java:192)
        at android.view.View.performClick(View.java:5637)
        at android.view.View$PerformClick.run(View.java:22429)
        at android.os.Handler.handleCallback(Handler.java:751)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6119)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
```

**Comment by deezy136** [comment]
Can I work on this issue?

**Comment by mikcox** [comment]
This was fixed with PR #9.
